src/lib/containers/RssFeed.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import rssParser from 'rss-parser'
+
+type RssState = {
+  rssFeed: rssParser.Output,
+  isLoadingError: boolean
+}
+
+export type RssFeedProps = {
+  url: string
+}
+
+export default class RssFeed extends React.Component<RssFeedProps, RssState> {
```

Comment:
I believe this class needs to get exported in the main index.ts file, I don't see that anywhere in this PR?